docs/tutorial-react-native/connecting.mdx

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 2
+---
+
+# Connecting
+
+How to connect to Fishjam Cloud
+
+## Getting URL and token
+
+In order to connect, you need to retrieve URL (multimedia server address) and token (token that will authenticate you in your Room)
+
+### Using sandbox app
+
+Once you get your account on [Fishjam Cloud](https://fishjam.io), you will have access to Sandbox App.
+This app comes with preconfigured test service called Room Manager. This is basically service that will
+create Room, add your app as Room's Participant and return token required to use that Room.
+
+To use that simply call `fetch`:
+```ts
+const response = await fetch(
+  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/*ROOM_NAME*/users/*USER_NAME*`,
+);
+
+const {url, token} = await response.json();
+```
```

Comment:
**suggestion:** 
I think it's a bit too complicated and too much detail for a basic tutorial. Let's trim it down to simple steps. 


1. Get your account on [Fishjam Cloud](https://fishjam.io)
2. Create a Room with a User
```ts
const response = await fetch(
  `https://fishjam.io/api/v1/connect/*YOUR_TOKEN*/room-manager/*ROOM_NAME*/users/*USER_NAME*`,
);

const {url, token} = await response.json();
```